tests/integration/components/donations/donation-progress-test.js

```diff
@@ -0,0 +1,34 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('donations/donation-progress', 'Integration | Component | donations/donation progress', {
+  integration: true
+});
+
+test('it renders proper information', function(assert) {
+  assert.expect(4);
+
+  let mockGoal = { amount: 1000, description: 'Lorem ipsum' };
+  this.set('donationGoal', mockGoal);
+
+  this.render(hbs`{{donations/donation-progress donationGoal=donationGoal amountDonated=500}}`);
+
+  assert.equal(this.$('.amount .value').text().trim(), '$500', 'Correct amount is rendered');
+  assert.equal(this.$('.percentage .value').text().trim(), '50%', 'Correct percentage is rendered');
+  assert.equal(this.$('.progress-bar').length, 1, 'Progress bar component is rendered');
+
+  assert.equal(this.$('.description').text().trim(), mockGoal.description, 'Goal description is rendered');
+});
+
+test('it renders decimal values if there are any', function(assert) {
+  assert.expect(3);
+
+  let mockGoal = { amount: 1000, description: 'Lorem ipsum' };
+  this.set('donationGoal', mockGoal);
+
+  this.render(hbs`{{donations/donation-progress donationGoal=donationGoal amountDonated=500.50}}`);
+
+  assert.equal(this.$('.amount .value').text().trim(), '$500.50', 'Correct amount is rendered');
+  assert.equal(this.$('.percentage .value').text().trim(), '50.05%', 'Correct percentage is rendered');
+  assert.equal(this.$('.progress-bar').length, 1, 'Progress bar component is rendered');
```

Comment:
We already briefly mentioned this in another PR. We do not, currently, use page objects in any integration test. They are acceptance tests only. I get the advantage of using it, but it would be a precedent, so I'm not sure if it should be done here or as a separate PR. We would also have to consider if such page objects need a different structure, or if we can just reuse component page objects used in acceptance tests.